public/input_text_load_data.js

```diff
@@ -0,0 +1,104 @@
+
+/*takes a string of data returns an array of that data*/
```

Comment:
Let's get more descriptive with this -- takes a string containing a multipart/mixed response from MarkLogic and returns an array of objects representing physical documents.